docs/development/setup.md

```diff
@@ -102,4 +102,27 @@ To remove the APIs from your cluster, simply run
 make uninstall
 ```
 
+**Note** In case `make uninstall` got stuck while deleting `onmetal-system` namespace, first delete all resources
+present in `onmetal-system` namespace and then delete `onmetal-system` namespace using below command.
+
+```shell
+kubectl delete ns onmetal-system
+```
+
+## Troubleshooting
+
+* Docker buildkit should be enabled while doing `make install` or `make kind-install`.
+
+  Error: "the --mount option require BuildKit"
+
+  Solution: Refer https://docks.docker.com/go/buildkit to enable BuildKit
+
+* Build platform should be known.
+
+  Error: "failed to parse platform : "" is an invalid component of "": platform specifier component must
+  match "^[A-Za-z0-9_-]+$": invalid argument" "
+
+  Solution: Provide platform to Dockerfile as `BUILDPLATFORM` varible before doing `make install` or
```

Comment:
Usually, docker automatically determines `BUILDPLATFORM` if not specified, see
https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

Did you check if your docker installation is up-to-date?
This would also probably resolve the `--mount` option issue you're getting.
If so, a comment  about a least required version of `docker` might help.

Reply:
Yes sure Axel. I will remove these details and add Docker with version in requirements section.
I faced this issue with Docker version 20.10.17 while deploying first time but second time it ran without any issue.